### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader:Title of the page is not descriptive

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3967]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
4.Start the narrator.
5.Use  CAPS+T to read the title of the page
**Actual:**
Title of the page should be descriptive.
**Note:
Same issue is observed in below pages:**
1. RunningTab: Navigate to run button & hit enter --> Navigate to running tab & hit enter.
**Expected:**
Title of the page should be specified  in a  descriptive manner.
**UserImpact:**
User will not know now currently user is visiting which page.. 
**Attachment:**
[A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader13.pptx](https://github.com/jupyter/notebook/files/2400069/A11y_AzureNotebooks_Jupyter_FilesTab_ScreenReader13.pptx)

**Comment by takluyver** [comment]
Closed by #4718
